api/models/User.js

```diff
@@ -13,7 +13,6 @@ module.exports = {
     },
     encryptedPassword: {
       type: 'string',
-      required: true,
```

Comment:
@itsalaidbacklife This is one thing I was unsure of. Since we don't really have a password for discord. I'm unsure of what to do in this situation

Reply:
Yeah let's discuss. We want to make sure you either have a discord identity or a password, but there's a chicken and egg problem because the discord identity requires a user id which then won't have the password. So what do we do to prevent users from being created with no way to log in?